core/modules/layout/layout.admin.inc

```diff
@@ -314,7 +314,7 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
       ', array(':path' => $path))
       ->fetchAllKeyed();
     if (count($path_matches)) {
-      $path_match_text = t('This layout will override the default layout for existing page(s) (which are created by %page_callback).', array('%page_callback' => $path_matches[$path]));
+      $path_match_text = t('This layout will override the default layout for existing page(s).');
```

Comment:
Would it make sense to add something like "...that match this path pattern." here?

Reply:
Something like that. Technically, "pages" don't match a pattern, their paths do, so:

"This layout will override the default layout for existing page(s) whose paths match this pattern."